CHANGELOG.md

```diff
@@ -7,25 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Revert tenant switch and clean up giantswarm tenant.
+
 ## [0.19.0] - 2025-03-06
 
 ### Added
 
 - Add cleanup for Mimir Alertmanager anonymous tenant's configuration.
 - Add cleanup for Mimir Ruler anonymous tenant's rules.
 
-### Fixed
-
-- Fix default read tenant to anonymous to ensure grafana rules pages work until the tenant switch is released.
-
 ## [0.18.0] - 2025-03-05
 
 ### Changed
 
 - Use smaller dockerfile to reduce build time as ABS already generates the go binary.
 - Read metrics from both anonymous and giantswarm tenant at once.
 - Refactor hardcoded tenant values to prepare the switch from the anonymous to the giantswarm tenant.
-- Switch the alerting component from the anonymous to the giantswarm tenant.
```

Comment:
Those lines should stay

Reply:
The last one sure yes

Reply:
Both, because we are not changing what's in previous releases

Reply:
I was confused, sorry

Reply:
Closing this in favor of https://github.com/giantswarm/observability-operator/pull/292